app/controllers/games_controller.rb

```diff
@@ -22,7 +22,7 @@ def update
 
   def show
     @game = Game.includes(:pieces).find(params[:id])
-    @piece = Piece.find(params[:id])
+    #@piece = Piece.find(params[:id])
   end
```

Comment:
@ruishine what do you mean by drop and recreate database? Because I tried rake db:migrate if that's what you meant...